next.config.js

```diff
@@ -1,6 +1,7 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {
+const withLess = require('next-with-less')
```

Comment:
Why adding less here. Every other `rc-*` works without less. Is this really required here?